packages/runtime/src/helpers/flags.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * If this flag is enabled, we generate one function per API Route.
+ * We'll also use the "none" bundling strategy where we fully rely on Next.js' `.nft.json` files.
+ * This should lead to smaller bundle sizes at the same speed, but is still experimental.
+ *
+ * If disabled, we bundle all API Routes into a single function.
+ * This is fast, but can lead to large bundle sizes.
+ *
+ * Enabled by default. Can be disabled by passing NEXT_SPLIT_API_ROUTES=false.
+ */
+
+export const splitApiRoutes = (featureFlags: Record<string, unknown>): boolean => {
+  if (process.env.NEXT_SPLIT_API_ROUTES) {
+    return process.env.NEXT_SPLIT_API_ROUTES === 'true'
+  }
+  // default to true during testing, swap to false before merging
+  return typeof featureFlags.next_split_api_routes === 'boolean' ? featureFlags.next_split_api_routes : true
```

Comment:
Yes. I went ahead and created https://app.devcycle.com/o/org_dhCAK5hN1u5tXdf9/p/shared/features/6458da4f1407798b5f6222ef. It's set to `false` for now, so if everything works correctly then the first part of the ternary should hit, and buildbot users should see `false`. We should also change the second part of the ternary to `false` right before merging, because CLI users won't see the flags. But I'd like to do this as the *very last thing* before merging, because the CI relies on the default case being `true` right now.

1. change the default value of this function to `false`. This is safe, but means that test code won't use the behaviour.
2. create a `next_split_api_routes` flag in DC and set it to `false`. That way, this codebase

Reply:
Shouldn't the tests then set the `NEXT_SPLIT_API_ROUTES` env var? Can we write the correct code here now so that we don't forget it and merge as-is

Reply:
Oh, that sounds like a lovely idea. Implementing! :loading-plz:

Reply:
Implemented in [f9f726f](https://github.com/netlify/next-runtime/pull/2058/commits/f9f726f5f83df1776a765b0ecf4ab6647b83ee43). One problem i'm seeing is that our cypress tests are using a deployment made by Netlify Build, so i'll also have to update the env var in the Netlify UI. Or, now that I think about it, maybe I can add them to `netlify.toml` instead!

Reply:
done in https://github.com/netlify/next-runtime/pull/2058/commits/60892960da51fa577d70f70ec686bf2c3315955a